### PR TITLE
guac-session improvements

### PR DESCRIPTION
**Pull request by heck-gd**
This PR contains some minor improvements for guac-session.

### CODE REVIEW

**Comment by doomedraven** [comment]
hey, thank you, can you please enable option that maintainers can edit PR?

File: conf/web.conf [hunk 1]
```diff
@@ -168,6 +168,8 @@ username =
 password =
 guacd_host = localhost
 guacd_port = 4822
+# Server that exposes the VNC ports (e.g., your KVM host)
```
Comment (doomedraven) [inline conf/web.conf:171]: no need to edit those files, only .default, i can't remove that by myself as i don't have rights

File: web/guac/views.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ def index(request, task_id, session_data):
         if state[0] == 1:
             vmXml = dom.XMLDesc(0)
             root = ET.fromstring(vmXml)
-            graphics = root.find("./devices/graphics")
-            vncport = graphics.get("port")
+            graphics = root.find("./devices/graphics[@type=\"vnc\"]")
+            vncport = graphics.get("port") if graphics is not None else None
```
Comment (doomedraven) [inline web/guac/views.py:35]: if graphics else None, i can't remove that by myself as i don't have rights

`if X` does check if is not None/Bool

**Comment by doomedraven** [comment]
thanks

**Comment by heck-gd** [comment]
@doomedraven I couldn't find a way to enable the option you mentioned, possibly because I forked into an organization.

**Comment by doomedraven** [comment]
np, thanks for pushing requested changes

**Pull request merged**